scaleway/resource_object_bucket_test.go

```diff
@@ -323,19 +333,22 @@ func TestAccScalewayObjectBucket_Cors_Delete(t *testing.T) {
 }
 
 func TestAccScalewayObjectBucket_Cors_EmptyOrigin(t *testing.T) {
+	if !*UpdateCassettes {
+		t.Skip("Skipping ObjectStorage test as this kind of resource can't be deleted before 24h")
+	}
 	tt := NewTestTools(t)
 	defer tt.Cleanup()
-	bucketName := "test-acc-scaleway-object-bucket-cors-empty-origin"
-	const resourceName = "scaleway_object_bucket.bucket"
 
+	randName := sdkacctest.RandomWithPrefix("bucket")
+	bucketName := sdkacctest.RandomWithPrefix("test-acc-scaleway-object-bucket-cors-empty-origin")
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:          func() { testAccPreCheck(t) },
 		ProviderFactories: tt.ProviderFactories,
 		CheckDestroy:      testAccCheckScalewayObjectBucketDestroy(tt),
 		Steps: []resource.TestStep{
 			{
 				Config: fmt.Sprintf(`
-					resource "scaleway_object_bucket" "bucket" {
+					resource "scaleway_object_bucket" "%s" {
```

Comment:
Does that mean that all assertions will be managed with string substitution as well? Why is this change needed?